PlanningPoker.WebApi/Controllers/UsersController.cs

```diff
@@ -0,0 +1,82 @@
+namespace PlanningPoker.WebApi.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.EntityFrameworkCore;
+    using PlanningPoker.Services;
+    using PlanningPoker.Shared;
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UsersController : ControllerBase
+    {
+        private IUserRepository repository;
```

Comment:
Mayby set all repos in the different controllers to 'readonly', otherwise everything looks fine imo.